doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+   Because of this, it must implement the mandatory mechanism to discover providers and query the OTA Provider periodically about new software images using the mandatory polling mechanism.
+   However, it can also receive information about the OTA Providers from the commissioner during :ref:`ug_matter_overview_commissioning`.
+
+OTA Provider
+   The device that provides the image for the OTA update.
+   This role corresponds to the OTA Provider device type, which implements the server role for the OTA Software Update Provider cluster and the client role for the OTA Software Update Requestor cluster.
+   Any device on the fabric can assume this role, as long as it has access to the Internet or to the database with links to software images.
+
+   The OTA Provider learns about new software images using any custom mechanism provided by the device manufacturer.
+   It can also learn about new software updates from the :ref:`ug_matter_device_dcl`, if it supports querying the DCL.
+
+   After the OTA Provider learns about a new software image, it downloads it to its internal cache.
+   Then, once requested, it sends to the OTA Requestor using Matter's :ref:`ug_matter_overview_dfu_image_bdx`.
+
+   The OTA Provider can announce its presence to OTA Requestors on the fabric.
+   The OTA Requestor can also learn about the OTA Providers on the fabric during commissioning or through dynamic discovery.
+
+.. figure:: /images/matter_ota_roles.svg
+   :alt: Matter OTA roles
+
+   Matter OTA roles
+
+The following parties can also be seen as participating in the process:
+
+Image Server
+   Provides the OTA image to the OTA Provider.
+   This is usually a server set up by the vendor who creates the software image, but the image can also be provided from a centralized source, such as :ref:`ug_matter_device_dcl`.
+
+User
+   Provides approvals for the OTA process.
+   Some OTA Requestors can directly ask the user for approval for applying an OTA update, for example using GUI or a network hub.
+   If asking directly is not possible, the approval request can be delegated to OTA Providers.
+
+.. figure:: /images/matter_ota_all_parties.svg
+   :alt: Parties to the Matter OTA process
+
+   Parties to the Matter OTA process
+
+.. _ug_matter_overview_dfu_reqs:
+
+Matter OTA requirements
+***********************
+
+For the Matter OTA process to happen, the following conditions must be met:
+
+* Both the OTA Requestor and OTA Provider must be members of the same fabric.
+* The OTA Requestor must be aware of the OTA Provider.
+* A source of software image must be available to the OTA Provider, for example DCL.
+* The software image must have a newer version than the firmware currently in use on the OTA Requestor.
+* The OTA Requestor must query the OTA Provider periodically about new software images using the polling mechanism.
+
+.. _ug_matter_overview_dfu_process:
+
+Matter OTA process
+******************
```

Comment:
I'm missing somewhere information, that image downloading process of Matter OTA happens in the background. So normal device's operation stay as they are.